docker/docker-compose.yml

```diff
@@ -9,12 +9,11 @@ services:
             args:
                 APP_CONFIG: ./docker/conf/app.config.sample
                 OSM_FILE: ./docker/data/heidelberg.osm.gz
+                JAVA_OPTS: -Dhttp.proxyHost=10.64.1.100 -Dhttp.proxyPort=80 -Dhttps.proxyHost=10.64.1.100 -Dhttps.proxyPort=443 -Djava.awt.headless=true -server -XX:TargetSurvivorRatio=75 -XX:SurvivorRatio=64 -XX:MaxTenuringThreshold=3 -XX:+UseConcMarkSweepGC -XX:+UseParNewGC -XX:ParallelGCThreads=4 -Xms1g -Xmx2g
```

Comment:
are the proxy host and port attributes required?

Reply:
Whoops - no, I left that in by mistake. Sorry about that.